jetty-util/src/main/java/org/eclipse/jetty/util/Jetty.java

```diff
@@ -18,14 +18,50 @@
 
 package org.eclipse.jetty.util;
 
+import org.eclipse.jetty.util.log.Log;
+import org.eclipse.jetty.util.log.Logger;
+
+import java.io.InputStream;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Properties;
+
 public class Jetty
 {
+    private static final Logger LOG = Log.getLogger( Jetty.class);
+
     public static final String VERSION;
     public static final String POWERED_BY;
     public static final boolean STABLE;
+    public static final String GIT_HASH;
+
+    /**
+     * a formatted build timestamp with pattern yyyy-MM-dd'T'HH:mm:ssXXX
+     */
+    public static final String BUILD_TIMESTAMP;
+    private static final Properties __buildProperties = new Properties( );
 
     static
     {
+        try
+        {
+            try (InputStream inputStream = //
+                     Jetty.class.getResourceAsStream( "/org/eclipse/jetty/version/build.properties" ))
+            {
+                __buildProperties.load( inputStream );
+            }
+        }
+        catch ( Exception e )
+        {
+            LOG.ignore( e );
+        }
+
+        GIT_HASH = __buildProperties.getProperty( "buildNumber", "unknown" );
+        System.setProperty( "jetty.git.hash" , GIT_HASH );
+        BUILD_TIMESTAMP = formatTimestamp( __buildProperties.getProperty( "timestamp", "unknown" ));
```

Comment:
@olamy @gregw I understand this PR is already merged, but I can't help but think it would be slightly better if the timestamp format was defined within the `buildnumber-maven-plugin` configuration, instead of in code.  That way, you could avoid the `SimpleDateFormat` parsing entirely and treat the timestamp as a formatted string generated by the build.

For example, in the `buildnumber-maven-plugin` configuration within `jetty-util/pom.xml`, add:

```xml
<timestampFormat>{0,date,yyyy-MM-dd'T'HH:mm:ssXXX}</timestampFormat>
```

As such, the `formatTimestamp` method can be deleted and this line becomes a simple:

```java
BUILD_TIMESTAMP = __buildProperties.getProperty( "timestamp", "unknown" );
```

Reply:
I think the point of the formatTimestamp() is to adjust and present the timestamp in the local timezone?

Reply:
@joakime Good point, but I'd rather the build timestamp always be `UTC-0` for consistency.

Reply:
We should probably just settle on the String form being ISO-8601 instant format and be done with it.
We are close with the format currently.   
Just have to force UTC TimeZone to ensure it always uses "Z" for "XXX" formatting pattern.

Or just use `java.time.format.DateTimeFormatter.ISO_INSTANT`